server/middleware/middleware.go

```diff
@@ -1,33 +0,0 @@
-package middleware
-
-import (
-	"github.com/giantswarm/micrologger"
-
-	"github.com/giantswarm/azure-operator/service"
-)
-
-// Config represents the configuration used to create a middleware.
-type Config struct {
-	// Dependencies.
-	Logger  micrologger.Logger
-	Service *service.Service
-}
-
-// DefaultConfig provides a default configuration to create a new
-// middleware by best effort.
-func DefaultConfig() Config {
-	return Config{
-		// Dependencies.
-		Logger:  nil,
-		Service: nil,
-	}
-}
-
-// New creates a new configured middleware.
-func New(config Config) (*Middleware, error) {
-	return &Middleware{}, nil
-}
-
-// Middleware is middleware collection.
-type Middleware struct {
-}
```

Comment:
I removed the middleware as well as the endpoint. As both are not used.

Reply:
I think this will throw an error on daemon start because so far the middleware is expected to not be empty.